app/brave_settings_strings.grdp

```diff
@@ -901,6 +901,39 @@
   <message name="IDS_SETTINGS_ENABLE_WIDEVINE_DESC" desc="Text fragment for enabling widevine component. 'Widevine' is the name of a plugin and should not be translated.">
     Use Widevine digital rights management (DRM) component to view protected streaming video and other content in the browser.
   </message>
+  <message name="IDS_SETTINGS_MANAGE_EXTENSIONS_V2_LABEL" desc="The label of manage v2 extensions link in settings">
+    Manifest V2 extensions
+  </message>
+  <message name="IDS_SETTINGS_MANAGE_EXTENSIONS_V2_SUBLABEL" desc="The sub label of manage v2 extensions link in settings">
+    The following Manifest V2 extensions will be removed from the Chrome Web Store, but are still supported in Brave. Click to download and install.
```

Comment:
Should this be "toggle to download and install, or to uninstall"?

Reply:
cc: @rmcfadden3 Any suggestions on the text. Also, @boocmp We can finalize text with a follow-up. I wouldn't consider this blocking.

Reply:
Per Slack convo with @rebron , let's go with this:

`The following Manifest V2 extensions will be removed from the Chrome Web Store, but still be supported in Brave. Toggle on an individual extension to download / install it, or toggle off to uninstall / remove.`